pkg/detectors/copyscape/copyscape.go

```diff
@@ -0,0 +1,90 @@
+package copyscape
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+	"io/ioutil"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"copyscape"}) + `\b([0-9a-z]{16})\b`)
+	idPat = regexp.MustCompile(detectors.PrefixRegex([]string{"copyscapeId"}) + `\b([0-9a-zA-Z._-]{4,22})\b`)
```

Comment:
could you please  `copyscape` instead of `copyscapeId`. Also can you verify that `.` works? I tested that it only allows letters, numbers, or underscores.